genomedata/__init__.py

```diff
@@ -403,11 +332,11 @@ def format_version(self):
         None means there are no chromosomes in it already.
         """
         assert self.isopen
-        if self._isfile:
-            try:
-                return self._file_attrs.genomedata_format_version
-            except AttributeError:
-                pass
+        # Get the Genomedata format version from the HDF5 file configurations
```

Comment:
In that specific case, it would depend on a `_format_version` attribute to be defined on a sub classed chromosome. This is likely to end up as `None`.

Reply:
I've fixed a minor issue in this case for the genomedata directory logic.